packages/client/package.json

```diff
@@ -18,16 +18,16 @@
   },
   "dependencies": {
     "@apollo/client": "^3.5.8",
-    "@babel/runtime": "^7.17.2",
+    "@babel/runtime": "^7.17.8",
     "@react-navigation/native": "^6.0.7",
     "@react-navigation/native-stack": "^6.3.0",
     "@react-navigation/stack": "^6.1.1",
-    "expo": "~44.0.0",
     "expo-constants": "~13.0.1",
     "expo-location": "~14.0.1",
     "expo-status-bar": "~1.2.0",
     "graphql": "^16.3.0",
     "jest-expo": "^44.0.1",
+    "ngrok": "^4.3.1",
```

Comment:
This is not a necessary dependency to install

Reply:
Ok